src/main/java/me/partlysanestudios/partlysaneskies/CustomMainMenu.java

```diff
@@ -460,6 +460,11 @@ public static void setMainMenuInfo(Request request) {
             e.printStackTrace();
             return;
         }
+        try {
+            Prank.Companion.setPrankKillSwitch(object.get("prank_sound").getAsBoolean());
+        } catch(Exception e){
```

Comment:
``(NullPointerException | IllegalStateException e)`` I thought this originally but ig that works